apps/emqx_bridge/src/emqx_bridge_v2.erl

```diff
@@ -750,7 +750,7 @@ create_dry_run(ConfRootKey, Type, Conf0) ->
             not_found ->
                 {error, iolist_to_binary(io_lib:format("Connector ~p not found", [ConnectorName]))};
             ConnectorRawConf ->
-                create_dry_run_helper(Type, ConnectorRawConf, Conf1)
+                create_dry_run_helper(ConfRootKey, Type, ConnectorRawConf, Conf1)
```

Comment:
Default dry_run is `actions`, we should use the confRootKey to do sources_probe.